Tests/UnitTests/Plugins/DD4hep/DD4hepCylindricalDetectorTests.cpp

```diff
@@ -318,11 +318,12 @@ BOOST_AUTO_TEST_CASE(DD4hepCylidricalDetectorExplicit) {
       bpCfg, Acts::getDefaultLogger("DD4hepBlueprint", Acts::Logging::VERBOSE));
   auto dd4hepBlueprint = bp.create(bpCache, world);
 
-  // We should have 5 store entries now
+  // We should have 6 store entries now
+  // 1 : beam pipe (empty)
   // 1 : endcap
   // 2 : barrel
   // 1 : endcap
-  BOOST_TEST(bpCache.dd4hepStore.size(), 5u);
+  BOOST_CHECK(bpCache.dd4hepStore.size() == 6u);
```

Comment:
Not critical, but you could do 

```suggestion
  BOOST_CHECK_EQUAL(bpCache.dd4hepStore.size(), 6u);
```

which gives you separate printouts of the arguments in the error case.

Reply:
Ok, I see - I can do a cleanup of those throughout then.